cmd/yggctl/actions.go

```diff
@@ -209,3 +188,10 @@ func generateMessage(messageType, responseTo, directive, content string, metadat
 		return nil, fmt.Errorf("unsupported message type: %v", messageType)
 	}
 }
+
+func connectBus() (*dbus.Conn, error) {
+	if os.Getenv("DBUS_SESSION_BUS_ADDRESS") != "" && os.Geteuid() > 0 {
```

Comment:
The refactoring is good, but we are loosing chance to print information what bus we tried to connect to. When it is not possible to connect to bus, then we do not important information... Was it session bus or was it system bus? We do not know... I have following proposal:

This block of code is duplicated after all usages of `connectBus()`.:

```go
if err != nil {
	return cli.Exit(fmt.Errorf("cannot connect to bus: %w", err), 1)
}
```

When `error` returned from `connectBus()` is not `nil`, then we can return the error little bit modified. There would be two variants of this error:
1. `fmt.Errorf("cannot connect to system bus: %w", err)`
2. `fmt.Errorf("cannot connect to session %s bus: %w", os.Getenv("DBUS_SESSION_BUS_ADDRESS"), err)`

Reply:
Hm. Yea. I can rework the `connectBus` function to return the bus address in the error message.

Reply:
I rewrote the `connectBus` function. I hope it is still explicit, but doesn't
lose the bus address information.